integration/update/jenkins-deployment-labels/input/.ship/state.json

```diff
@@ -0,0 +1,26 @@
+    "kustomize": {
```

Comment:
based on the title, I would have expected to see a patch here to add a label to a deployment?

Reply:
That sounds like something I'd like too